src/js/components/SingleImage.vue

```diff
@@ -1,20 +1,21 @@
+// TODO move menu into its own component
```

Comment:
Suggest putting these review comments into the script block. I miss them when they're above the template.